### PR TITLE
Reactivate Windows CI

### PR DESCRIPTION
**Pull request by irux**

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -1,26 +1,18 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```
Comment (disrupted) [inline poetry.lock:1]: The lockfile is broken, please update your version of Poetry and re-run `poetry lock --no-update`

**Review by disrupted — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -26,14 +26,14 @@ jobs:
       - name: Install Python and set up Poetry
         uses: bakdata/ci-templates/actions/python-setup-poetry@v1.5.3
         with:
-          poetry-version: "1.5.1"
+          poetry-version: "1.4.0"
```
Comment (disrupted) [inline .github/workflows/ci.yaml:29]: is downgrading still necessary?

Comment (irux) [reply .github/workflows/ci.yaml:29]: yes

Comment (disrupted) [reply .github/workflows/ci.yaml:29]: could you link to the issue you created on the Poetry repo, please?

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -26,14 +26,14 @@ jobs:
...
           python-version: ${{ matrix.python-version }}
 
       - name: Install dependencies
         run: poetry install --no-interaction
 
       - name: Lint (flake8)
-        run: poetry run pre-commit run flake8 --all-files --show-diff-on-failure
+        run: poetry run pre-commit run flake8 --verbose --all-files --show-diff-on-failure
```
Comment (disrupted) [inline .github/workflows/ci.yaml:36]: still necessary?

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (irux) [reply .github/workflows/ci.yaml]: don't ask me why but now it pass with 1.5.1 xD

**Review by disrupted — APPROVED** [review]
LGTM

**Review by raminqaf — APPROVED** [review]

**Pull request merged**